.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
```

Comment:
I find tools like dependably to be super annoying so I don't want to add this config. Thanks though.

Reply:
It will keep all dependencies + actions (which will be missed if we don't look at them manually) updated to their latest version in all repos with this change, that's why I added it

Reply:
Yeah, I know a lot of people like that, but when you have as many repositories as I do it gets to be super annoying and I'm of the opinion that if it's working then there's no point in upgrading. I typically upgrade stuff when there's a need or when I come in to update something.

Reply:
Should we keep the `github-actions` part then, as that's easily overlooked and really manual work?

Reply:
Nah, like I said, if it's working then I don't feel super motivated to upgrade things and find robots bugging me about out-of-date-but-working-dependencies to be really annoying.